frontend/components/surplus/SurplusAuctionTransactionTable.vue

```diff
@@ -0,0 +1,97 @@
+<template>
```

Comment:
Please remove the Table file & its story file since we aren't adding it in this PR anymore.